src/user.ts

```diff
@@ -29,7 +29,7 @@ import { SecureBuffer } from './secureBuffer';
 import { SfdxError } from './sfdxError';
 import { sfdc } from './util/sfdc';
 
-const PASSWORD_LENGTH = 10;
+const PASSWORD_LENGTH = 13;
```

Comment:
when generating a password we were using `PASSWORD_LENGTH - ALL.length`, with `const ALL = [LOWER, UPPER, NUMBERS, SYMBOLS.join('')];` this was resulting in a password length of 6 with the minimum required being 8 and the server was throwing an error